src/View/Components/IncidentTimeline.php

```diff
@@ -43,21 +48,32 @@ public function render(): View
      */
     private function incidents(Carbon $startDate, Carbon $endDate, bool $onlyDisruptedDays = false): Collection
     {
+        $appSettings = $this->appSettings;
```

Comment:
Since we need to pass this variable to lower scopes, I'd probably avoid the temporary variable and use the property within the closure.